lib/Dialect/Mgmt/Transforms/BUILD

```diff
@@ -0,0 +1,56 @@
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library")
+
+package(
+    default_applicable_licenses = ["@heir//:license"],
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "Transforms",
+    hdrs = ["Passes.h"],
+    deps = [
+        ":AnnotateMgmt",
+        ":pass_inc_gen",
+        "@heir//lib/Dialect/Mgmt/IR:Dialect",
+        "@llvm-project//mlir:IR",
+    ],
+)
+
+cc_library(
+    name = "AnnotateMgmt",
+    srcs = ["AnnotateMgmt.cpp"],
+    hdrs = ["AnnotateMgmt.h"],
+    deps = [
+        ":pass_inc_gen",
+        "@heir//lib/Analysis/DimensionAnalysis",
+        "@heir//lib/Analysis/LevelAnalysis",
+        "@heir//lib/Analysis/SecretnessAnalysis",
+        "@heir//lib/Dialect/Mgmt/IR:Dialect",
+        "@llvm-project//mlir:IR",
+        "@llvm-project//mlir:Pass",
+        "@llvm-project//mlir:Transforms",
+    ],
+)
+
+gentbl_cc_library(
+    name = "pass_inc_gen",
+    tbl_outs = [
+        (
+            [
+                "-gen-pass-decls",
+                "-name=Mgmt",
+            ],
+            "Passes.h.inc",
+        ),
+        (
+            ["-gen-pass-doc"],
+            "MgmtPasses.md",
+        ),
+    ],
+    tblgen = "@llvm-project//mlir:mlir-tblgen",
+    td_file = "Passes.td",
+    deps = [
+        "@llvm-project//mlir:OpBaseTdFiles",
+        "@llvm-project//mlir:PassBaseTdFiles",
+    ],
+)
```

Comment:
```suggestion
load("@heir//lib/Transforms:transforms.bzl", "add_heir_transforms")

package(
    default_applicable_licenses = ["@heir//:license"],
    default_visibility = ["//visibility:public"],
)

cc_library(
    name = "Transforms",
    hdrs = ["Passes.h"],
    deps = [
        ":AnnotateMgmt",
        ":pass_inc_gen",
        "@heir//lib/Dialect/Mgmt/IR:Dialect",
        "@llvm-project//mlir:IR",
    ],
)

cc_library(
    name = "AnnotateMgmt",
    srcs = ["AnnotateMgmt.cpp"],
    hdrs = ["AnnotateMgmt.h"],
    deps = [
        ":pass_inc_gen",
        "@heir//lib/Analysis/DimensionAnalysis",
        "@heir//lib/Analysis/LevelAnalysis",
        "@heir//lib/Analysis/SecretnessAnalysis",
        "@heir//lib/Dialect/Mgmt/IR:Dialect",
        "@llvm-project//mlir:IR",
        "@llvm-project//mlir:Pass",
        "@llvm-project//mlir:Transforms",
    ],
)

add_heir_transforms(
    generated_target_name = "pass_inc_gen",
    pass_name = "Mgmt",
    td_file = "Passes.td",
)
```

In an effort to simplify some of the bazel stuff, we added this add_heir_transforms macro. I think this usage will work as a drop-in replacement. (Also, did you use the template generator to generate this? If so, I should update that to use the macro).

Reply:
> Also, did you use the template generator to generate this? If so, I should update that to use the macro

I used the template generator. That should be updated.